tests/console/cups.pm

```diff
@@ -14,7 +14,8 @@ use base 'consoletest';
 use strict;
 use warnings;
 use testapi;
-use utils;
+use utils qw(systemctl zypper_call);
+use Utils::Systemd 'disable_and_stop_service';
```

Comment:
This module is exporting by default and it only has one thing so far, I guess it's safe to simply `use Utils::Systemd;`. However I leave this to you :) It's not really harmful to just import what we need :)

Reply:
I prefer to have the function list explicitly. It is easier to remove the use statement when we remove a function call.